lib/core/src/Cardano/Wallet/DB/SqliteTypes.hs

```diff
@@ -0,0 +1,242 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
What orphans do we have in this module?

Reply:
they are servicing mainly what is needed in `Cardano.Wallet.DB.Sqlite` module. As persistent uses template haskell and instances  are needed at compile time, we decided to put them here